aten/src/ATen/native/quantized/cpu/qrelu.cpp

```diff
@@ -113,7 +113,7 @@ Tensor& leaky_relu_out_quantized_cpu(Tensor& result, const Tensor& self,
   return result;
 }
 
-Tensor heaky_relu_quantized_cpu(const Tensor& self, Scalar negval) {
+Tensor leaky_relu_quantized_cpu(const Tensor& self, Scalar negval) {
```

Comment:
For my understanding: When is this called compared to QLeakyRelu::run?

Reply:
this is called by the original leaky_relu op that works on both float and quantized tensors and it does not need quantization parameter, QLeakyRelu::run is called by quantized::leaky_relu, which only works on quantized tensor and needs output quantization parameters